docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sysdis-monitor.md

```diff
@@ -0,0 +1,115 @@
+   * ![green check circle.png](/img/reuse/green-check-circle.png) A green circle with a check mark is shown when the field exists in the Fields table schema.
+   * ![orange exclamation point.png](/img/reuse/orange-exclamation-point.png) An orange triangle with an exclamation point is shown when the field doesn't exist in the Fields table schema. In this case, an option to automatically add the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo Logic that does not exist in the Fields schema it is ignored, known as dropped.
+7. In **Base URL**, choose the URL where your **Sysdig** account is located. To know your base URL, see [Base URL](#base-url) section.
+8. In **API Key**, authenticate your account by entering your secret API key. You can access your API key or generate a new one from **User Event API Management Console**, see [API Token](#api-token) section
+9. When you are finished configuring the Source, click **Save**.
+
+### Error types
+
+When Sumo Logic detects an issue, it is tracked by Health Events. The following table shows three possible error types. It tells the reason for the error, if the source attempts to retry, and the name of the event log in the Health Event Index.
+
+| Type | Reason | Retries | Retry Behavior | Health Event Name |
+|:--|:--|:--|:--|:--|
+| ThirdPartyConfig  | Normally due to an invalid configuration. You'll need to review your Source configuration and make an update. | No retries are attempted until the Source is updated. | Not applicable                                                    | ThirdPartyConfigError  |
+| ThirdPartyGeneric | Normally due to an error communicating with the third party service APIs.                                     | Yes                                                   | The Source will retry for up to 90 minutes, after which it quits.                               | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs.                                     | Yes                                                   | The Source will retry for up to 90 minutes, after which it quits.                               | FirstPartyGenericError |
+
+### JSON configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the Collector Management API. See [how to use JSON to configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Required | Description | Access |
+|--|--|--|--|--|
+| `config` | JSON Object  | Yes | Contains the [configuration-parameters](#config-parameters) of the Source. |
+| `schemaRef` | JSON Object  | Yes | Use `{"type":"Sysdig Monitor Source"}` for Sysdig Monitor Source. | not modifiable |
+| `sourceType` | String | Yes | Use `Universal` for Sysdig Monitor Source. | not modifiable |
+
+### Config Parameters
+
+| Parameter | Type | Required | Description | Access |
+|--|--|--|--|--|
```

Comment:
```suggestion
|:--|:--|:--|:--|:--|
```

Reply:
These colons are just to fix the alignment. In this case, these columns would work without these colons as well. So no harm even if we do not apply these colons here.